README.md

```diff
@@ -70,16 +70,15 @@ doSometing(catalogs);
 
 ### Compatibility
 
-The `platform-client` package is built on top of the `fetch` API and some related features such as the `AbortController` which are not entirely supported by all JavaScript runtime environments. Consequently, we recommend including the following list of polyfills to your project before using it in production.
-
--   [Polyfill for `fetch`](https://github.com/matthew-andrews/isomorphic-fetch)
--   [Polyfill for `AbortController`](https://github.com/mo/abortcontroller-polyfill)
+The `platform-client` package is built on top of the `fetch` API and is not entirely supported by all JavaScript runtime environments.
+Consequently, for Node, we recommend using Node 18 or `undici` to polyfill `fetch` globally
 
 #### Node.js example
 
 ```js
-require('isomorphic-fetch');
-require('abortcontroller-polyfill');
+if (!global['fetch']) {
```

Comment:
So if I understand correctly, if i'm rocking node 16 locally because, I can still use platfom? 🤔

Reply:
If you use the experimental flag --experimental-fetch yep, or if you set the globals yourself

Reply:
<img src="https://media3.giphy.com/media/vbFG3WKT8ysK3vNP1E/giphy.gif"/>